### DISCUSSION TITLE
Error with mobile mode and headless mode together

### DISCUSSION THREAD

**Discussion by joako233** [seleniumbase/SeleniumBase#2730] [Q&A] [answered]
Hi! I encountered this error while using the mobile emulator mode with headless mode enabled. Does anyone have any insights on how to troubleshoot or resolve it?: **[0429/210003.114:ERROR:webgpu_decoder_impl.cc(1636)] Failed to query ID3D11Device from ANGLE.**
![image](https://github.com/seleniumbase/SeleniumBase/assets/141453948/8e7424b2-e8e0-4617-a9f7-c6b9ee6ab69d)

**Comment by mdmintz** [comment, marked as answer]
Use the newer headless mode: `headless2=True`.

**Comment by joako233** [comment]
Thanks!!
